plugins/katello/3.13/release_notes/release_notes.md

```diff
@@ -6,11 +6,62 @@ version: 3.13
 
 # Katello {{ page.version }} Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.13/CHANGELOG.md).
 
 ## Features
 
-## Deprecation Warnings
+### System Purpose Support
+
+Katello 3.13 provides setting system purpose attributes on Activation Keys in the UI. Additionally, the Hammer CLI suports setting System Purpose attrbutes on Activation Keys and Content Hosts.
+
+### HTTP Proxy Support
+
+Katello 3.13 introduces support for HTTP proxies for content syncing. Repositories may be assigned an HTTP proxy policy: a global default, no proxy, o specify a proxy. HTTP proxy policies can be applied through bulk actions at the product level.
+
+### Content Views
+
+Content View descriptions may now be updated via the Hammer CLI.
+
+### SRPM Support
+
+Katello 3.13 provides the ability to upload SRPM's via the API and through Hammer and katello to correctly identify the content as a source RPM, previous behavior we would get an error from Pulp.
```

Comment:
s/SRPM's/SRPMs

s/katello/Katello

I think we can also remove the 'previous behavior' part since we can assume it didn't work in one way or another